src/exo/platforms/gemmini.py

```diff
@@ -2,143 +2,172 @@
 
 from exo import proc, instr, DRAM, config, QAST
 from exo.libs.memories import GEMM_SCRATCH, GEMM_ACCUM
+from exo.stdlib.scheduling import *
```

Comment:
Gemmini changes are great, thank you for looking into this!
I'll ask Jennifer to test this on AWS before merging alongside with gemmini-rocc-tests stuff